### PR TITLE
DOCSP 31207 Clarify mongosync reverse documentation unique index limitation

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Clarify mongosync reverse documentation unique index limitation with steps on resending all nodes in a source cluster. 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-31207/reference/api/reverse/#validate-unique-indexes

## JIRA

https://jira.mongodb.org/browse/DOCSP-31207

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=654bdb2d5b686d6bd975ac96

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/api/reverse.txt [hunk 1]
```diff
@@ -63,16 +63,31 @@ Clusters that began with MongoDB 4.2 or older and were since
...
+   .. step:: Step down the primary node and step up one of the secondary nodes.
+   
+   .. step:: Resync the old primary node to the new primary node. 
+
+Alternatively, to avoid resyncing, you can use the
+:method:`db.collection.validate` method with ``full = false`` on each
+collection with unique indexes to determine whether each collection
+contains improperly formatted unique indexes. 
 
 .. code-block:: javascript
```
Comment (ajhuh-mdb) [inline source/reference/api/reverse.txt:85]: [suggestion] This code block here seems a bit random and not super useful. We might want to remove this.

File: source/reference/api/reverse.txt [hunk 2]
```diff
@@ -63,16 +63,31 @@ Clusters that began with MongoDB 4.2 or older and were since
 upgraded may include unique indexes that are not properly formatted.
 
 To correct indexes, you can :ref:`resync <resync-replica-member>` all nodes
-in the original source cluster.  If you don't want to resync the cluster, you
-can use the :method:`db.collection.validate` method on each collection to 
-determine whether it contains improperly formatted unique indexes.
+in the original source cluster.  To resync all nodes: 
+
+.. procedure::
```
Comment (ajhuh-mdb) [inline source/reference/api/reverse.txt:68]: [praise] Really like the use of procedures here!

**Review by ajhuh-mdb — APPROVED** [review]
LGTM with one small suggestion! Thanks @ltran-mdb2

File: source/reference/api/reverse.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/reference/api/reverse.txt]: Done!

**Comment by ltran-mdb2** [comment]
Hi @rohshar ! Would you be able to give a tech review to this PR on the mongosync reverse documentation?

File: source/reference/api/reverse.txt [hunk 3]
```diff
@@ -63,16 +63,26 @@ Clusters that began with MongoDB 4.2 or older and were since
...
-resync all of the nodes in the original source cluster before reversing sync.
+      For a tutorial on resyncing nodes, see :ref:`Resync a Member of a
+      Replica Set <resync-replica-member>`. 
+
+   .. step:: Step down the primary node and step up one of the secondary nodes.
+   
+   .. step:: Resync the old primary node to the new primary node. 
+
+Alternatively, to avoid resyncing, you can use the
+:method:`db.collection.validate` method with ``full = false`` on each
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/reference/api/reverse.txt:81]: Done!

File: source/reference/api/reverse.txt [hunk 4]
```diff
@@ -63,16 +63,26 @@ Clusters that began with MongoDB 4.2 or older and were since
...
+   .. step:: Resync all secondaries, one by one. 
 
-If the method returns a warning about the unique index, you must
-resync all of the nodes in the original source cluster before reversing sync.
+      For a tutorial on resyncing nodes, see :ref:`Resync a Member of a
+      Replica Set <resync-replica-member>`. 
+
+   .. step:: Step down the primary node and step up one of the secondary nodes.
+   
+   .. step:: Resync the old primary node to the new primary node.
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/reference/api/reverse.txt]: Done!

**Comment by ltran-mdb2** [comment]
Thanks for the feedback @rohshar ! Could you take another look when you get a chance?

**Review by rohshar — APPROVED** [review]

**Pull request merged**